packages/plugin-breadcrumbs-console/src/__tests__/index.test.ts

```diff
@@ -112,4 +112,25 @@ describe("BreadcrumbsConsolePlugin", () => {
       })
     })
   })
+
+  describe("when console function is called with a circular argument", () => {
```

Comment:
It would be neat to add a test where an object _without_ a circular argument is called -- that is, one where the branch of `serializeValue` that calls `JSON.stringify` is executed, but that does not result in a failure to serialize.

Reply:
Yes! I've added a test for it.